### ISSUE TITLE
'Enter' key is interpreted as a new line instead of the end of the prompt.

### ISSUE THREAD

**Issue by slimjimo** [vincelwt/chatgpt-mac#79]

**Comment by ertoorule** [comment]
It can also be a key combination like shift+enter

**Comment by slimjimo** [comment]
Yep, both the enter key and shift+enter are interpreted as starting a new line. No key/key combination on my keyboard allows for me to signal the end of the prompt, meaning I have to manually click the send button.

**Comment by aaajhs** [comment]
I'm seeing the same behaviour. This was not the case just a few days ago, which leads me to think this is a bug.

**Comment by jnsw** [comment]
what do we do now?

**Comment by lukasong** [comment]
Wait for the developer to fix it, although this project seems abandoned as of now. Maybe a fork or PR can do the trick.

**Comment by olerap** [comment]
For now you can use tab to highlight the send button and then press enter.

**Comment by vincentor** [comment]
this annoys me too, hope to fix it soon.

**Comment by eddiewang** [comment]
to save someone some time just throw this in right at the top of `before-input-event` event:

```
        if (input.type !== "keyDown" && key == "Enter") {
          // press main > form > button
          contents.executeJavaScript(
            `document.querySelector('main form button').click()`
          );
        }
```

This is due to the ChatGPT mobile wanting to ensure Enter keypresses default to multi-line input.

**Comment by AccessRetrieved** [comment]
> to save someone some time just throw this in right at the top of `before-input-event` event:
> 
> ```
>         if (input.type !== "keyDown" && key == "Enter") {
>           // press main > form > button
>           contents.executeJavaScript(
>             `document.querySelector('main form button').click()`
>           );
>         }
> ```
> 
> This is due to the ChatGPT mobile wanting to ensure Enter keypresses default to multi-line input.

Put this in the `index.js` in the `before-input-event` and delete the `forge.config.js` file. It generates an error while running locally. 

I just created an automator application that runs the app with `npm run start` and changed the app icon. Works perfectly now


https://user-images.githubusercontent.com/56012931/220539070-c9041497-63ea-487b-b74c-671a87431bca.mp4

**Comment by AccessRetrieved** [comment]
> > to save someone some time just throw this in right at the top of `before-input-event` event:
> > ```
> >         if (input.type !== "keyDown" && key == "Enter") {
> >           // press main > form > button
> >           contents.executeJavaScript(
> >             `document.querySelector('main form button').click()`
> >           );
> >         }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This is due to the ChatGPT mobile wanting to ensure Enter keypresses default to multi-line input.
> 
> Where? Can this be done by the end user?

Download the repo and run it locally.

**Comment by eddiewang** [comment]
@paulmoons I created a MR for this here #92 - if the maintainer merges and rebuilds, this should fix any issues you have with the `Enter` key.

Hopefully this helps!

**Comment by paulmoons** [comment]
> @paulmoons I created a MR for this here #92 - if the maintainer merges and rebuilds, this should fix any issues you have with the `Enter` key.
> 
> Hopefully this helps!

Thanks!

**Comment by Midicii** [comment]
No Code Solution: 

https://user-images.githubusercontent.com/56598150/221528888-f3e42203-22fe-4328-ab3b-61aa3eac931b.mov

**Comment by paulmoons** [comment]
> No Code Solution:

Yeah, thanks. It's a nice solution but it needs to be applied every time the app starts as the screen size isn't remembered. Thanks though!

**Comment by ssapsu** [comment]
> No Code Solution:

you're really a genius.

**Comment by gmshrek** [comment]
[ChatBox](https://github.com/gmshrek/chatbox )

It supports custom enable and send shortcut keys.

**Comment by paulmoons** [comment]
Solution is here folks: https://goodsnooze.gumroad.com/l/menugpt

**Comment by katzkuma** [comment]
There's another no-code solution:
I've been using this key combination like Command + Enter. It works for me.
